src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -10,10 +10,27 @@
           <td>{{ hostsCount }}</td>
         </tr>
         <tr>
-          <td i18n
-              class="bold">Storage Capacity</td>
-          <td><span i18n>Number of devices: {{ totalDevices }}. Raw capacity:
-            {{ totalCapacity | dimlessBinary }}.</span></td>
+          <td>
+          <dl>
+            <dt>
+              <p>
+                Storage Capacity
+              </p>
+            </dt>
+            <dd>
+              <p>
+                Number of devices
+              </p>
+            </dd>
+            <dd>
+              <p>
+                Raw capacity
+              </p>
+            </dd>
+          </dl>
+        </td>
+          <td class="indent"><span i18n><p>{{ totalDevices }}</p><p>
```

Comment:
```suggestion
          <td class="indent"><p>{{ totalDevices }}</p><p>
```
It seems there is no plain text to translate here.